stdlib/fun.ml

```diff
@@ -20,6 +20,12 @@ let negate p v = not (p v)
 
 exception Finally_raised of exn
 
+let _ = Printexc.register_printer (function
```

Comment:
I'd use `let () = ...` it's clearer/safer.

Reply:
done.